packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -2605,7 +2603,7 @@ curl -v -X POST \
 
 > **Note:** This is a <ApiLifecycle access="deprecated" /> feature.
 
-Operations for just-in-time (JIT) provisioning or account linking with a `callout` action (webhook)
```

Comment:
Not sure the 'i' of 'in' should be capitalized, but don't know.

Reply:
in this instance, it should since it's a thing